### PR TITLE
Change business offering percentage fields to booleans.

### PR DESCRIPTION
**Pull request by neoeno**
See https://github.com/moneyadviceservice/mas-rad_core/pull/77

This is pretty much a straight search-and-replace, and I'd rather avoid cleaning up existing code without good reason — so bear that in mind when reviewing!

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -315,20 +315,19 @@ en:
         subheading: "Percentage of overall business income taken from the following types of advice:"
```
Comment (munckymagik) [inline config/locales/en.yml]: This string will need to be reworded.

Comment (neoeno) [inline config/locales/en.yml]: You're right, I meant to ask Mark about this. Will do

**Comment by munckymagik** [comment]
:+1:

**Comment by darokel** [comment]
:+1:

**Comment by darokel** [comment]
What's the context behind this change?

**Comment by munckymagik** [comment]
@darokel will answer in Slack for ease

**Pull request merged**